front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.spec.ts

```diff
@@ -52,5 +60,17 @@ describe('CreateF3XStep1Component', () => {
     component.form.controls['filing_frequency'].setValue('Q');
     component.form.controls['report_type_category'].setValue(F3xReportTypeCategories.SPECIAL);
     expect(component.form.controls['report_code'].value).toEqual(F3xReportCodes.TwelveP);
+    component.form.controls['report_type_category'].setValue(F3xReportTypeCategories.NON_ELECTION_YEAR);
+    expect(component.form.controls['report_code'].value).toEqual(F3xReportCodes.Q1);
+    component.form.controls['report_type_category'].setValue(undefined);
+    expect(component.form.controls['report_code'].value).toEqual(undefined);
+  });
+
+  it('#save should save a new f3x record', () => {
+    component.form.patchValue({ ...f3x });
+    component.save();
+    const req = httpTestingController.expectOne(`${environment.apiUrl}/f3x-summaries/`);
+    expect(req.request.method).toEqual('POST');
+    httpTestingController.verify();
   });
 });
```

Comment:
The unit tests are generating warnings/errors about unknown elements
![app-138-unit-test-errors](https://user-images.githubusercontent.com/2971785/166688260-2533d6bf-d892-44c3-9387-ca0d8224a137.png)
.